ci-operator/step-registry/azure/provision/vnet/azure-provision-vnet-commands.sh

```diff
@@ -32,7 +32,9 @@ function create_disconnected_network() {
     local nsg rg="$1" subnet_nsgs="$2"
     for nsg in $subnet_nsgs; do
         run_command "az network nsg rule create -g ${rg} --nsg-name '${nsg}' -n 'DenyInternet' --priority 1010 --access Deny --source-port-ranges '*' --source-address-prefixes 'VirtualNetwork' --destination-address-prefixes 'Internet' --destination-port-ranges '*' --direction Outbound"
-        run_command "az network nsg rule create -g ${rg} --nsg-name '${nsg}' -n 'AllowAzureCloud' --priority 1009 --access Allow --source-port-ranges '*' --source-address-prefixes 'VirtualNetwork' --destination-address-prefixes 'AzureCloud' --destination-port-ranges '*' --direction Outbound"
+        if [[ "${CLUSTER_TYPE}" != "azurestack" ]]; then
```

Comment:
Does that means ASH does nto need this nsg rule?

Reply:
This command failed to be executed on ASH. I checked there is no destination service tag named as "AzureCloud" on ASH WWT, it only exists on azure public cloud and azure gov cloud.
Seems no function impact without this rule on ASH, I checked that it is unreachable to connect from cluster to internet and cluster works well behind proxy.

Reply:
Got it.